error/runtime_spec.go

```diff
@@ -41,6 +41,73 @@ const (
 	// ReadonlyOnWindows represents the error code of readonly setting test on Windows
 	ReadonlyOnWindows
 
+	// MountInOrder represents the error code of order test of mount list
+	MountInOrder
+	// MountDestinationAbs represents the error code of absolute path test of mount destination
+	MountDestinationAbs
+	// MountNotNestedOnWindows represents the error code of nested mount point on Windows
+	MountNotNestedOnWindows
```

Comment:
I recommend consistent platform namespacing.  You have an `OnWindows` suffix here, but have other entries with a platform prefix (e.g. `WindowsPlatform`, `LinuxOomScoreAdj`, `PosixHookPathAbs`).  I recommend a template like:

    {platform, if platform-specific}{JSON path in camelCase}{something about what's being tested}

which would make this one `WindowsMountsDestinationNotNested`.